tests/test_poisson_fem_cube.py

```diff
@@ -147,7 +147,7 @@ def test_computeError(self):
 		from mozart.poisson.fem.cube import solve
 		from mozart.poisson.fem.cube import computeError
 		iter = 4
-		for degree in range(1,4):
+		for degree in range(1,2):
```

Comment:
The reason that @creamy11 decreased `degree` from 3 to 2. maybe its poor performance(too slow) test case. 
It takes 160 seconds on my desktop(quite fast).